asdf/tests/test_generic_io.py

```diff
@@ -821,3 +822,19 @@ def test_fsspec_http(httpserver):
         gf = generic_io.get_file(f)
         r = gf.read(len(ref))
         assert r == ref, (r, ref)
+
+
+@pytest.mark.skipif(
+    sys.platform.startswith("win"),
+    reason="Cannot test write file permissions on windows",
+)
+@pytest.mark.parametrize("umask", [0o22, 0o2, 0o0, 0o44, 0o66])
```

Comment:
Again can you do as I asked before in order document what each of these octal numbers mean?

Also, without figuring out these 5 values, does this cover all possible permutations of permissible file permissions?